package.json

```diff
@@ -2,16 +2,17 @@
     "type": "module",
     "private": true,
     "scripts": {
+        "build": "npm run db:pack",
         "db:unpack": "node tooling/unpack-macros.mjs",
         "db:pack": "node tooling/pack-macros.mjs",
         "new-effect": "node tooling/create-new-effect.mjs",
-        "test": "npm run test:packs && npm run test:js",
-        "test:js": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
-        "test:packs": "npm run db:pack && git diff --no-patch --exit-code packs/weaponfx.db && echo 'Pack test passed!'"
+        "test": "npm run test:js",
+        "test:js": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
     },
     "devDependencies": {
+        "@foundryvtt/foundryvtt-cli": "^1.0.2",
+        "chalk": "^5.3.0",
```

Comment:
`chalk` is a transitive dependency from `@foundryvtt/foundryvtt-cli`, so this is "free"